ci-operator/step-registry/cucushift/installer/rehearse/baremetalds/ipi/ovn/lvms/cucushift-installer-rehearse-baremetalds-ipi-ovn-lvms-workflow.yaml

```diff
@@ -13,5 +13,6 @@ workflow:
         VM_EXTRADISKS=true
         VM_EXTRADISKS_LIST="vda vdb"
         VM_EXTRADISKS_SIZE=100G
+      LVM_CLUSTER_TOLERATE_MASTER: true
```

Comment:
`LVM_CLUSTER_TOLERATE_MASTER` is on the same level of `DEVSCRIPTS_CONFIG`, not a sub env under `DEVSCRIPTS_CONFIG`, right ?

Reply:
Yeah, correct.